enforcer-parent/enforcer/src/main/resources/prometheus-jmx-config.yml

```diff
@@ -0,0 +1,42 @@
+# --------------------------------------------------------------------
+# Copyright (c) 2023, WSO2 Inc. (http://wso2.com).
```

Comment:
Copyright (c) 2023, WSO2 LLC.

It's better if we can update other places in the same PR.

Reply:
Fix [de37947](https://github.com/wso2/product-microgateway/pull/3428/commits/de379477a8a20379d29d4fa4a4e0bc48d3e0fb18)